backend/main/models/community_users.go

```diff
@@ -145,59 +145,22 @@ func GetCommunitiesForUser(db *s.Database, addr string, start, count int) ([]Use
 	err := pgxscan.Select(db.Context, db.Conn, &communities,
 		`
 		SELECT
-	  communities.*,
-	  a.roles
-		FROM communities
-		LEFT JOIN community_users ON community_users.community_id = communities.id
-		LEFT JOIN (
-				SELECT DISTINCT ON (community_users.community_id) community_users.*,
-						ARRAY(
-								SELECT community_users.user_type
-								FROM community_users
-								WHERE addr = $1
-								GROUP BY community_users.user_type
-						) as roles
-						FROM community_users
-						WHERE addr = $1
-		) a ON community_users.addr = a.addr
+		communities.*,
+		community_users.user_type as roles
+		  FROM communities
+		  LEFT JOIN community_users ON community_users.community_id = communities.id
```

Comment:
i believe this should be a regular `JOIN` rather than a `LEFT JOIN`.  This will select every community from the DB, and as this number grows, the result could get very large

Reply:
Fixed.